### ISSUE TITLE
Add Discord webhook integration

### ISSUE THREAD

**Issue by jezevec10** [bhainesva/jstris-leaderboard-changes#1]
Hi, I've installed your script and it seems to work very well. We would be interested in using it in Jstris discord to post daily reports about how the leadeboard changed.

Consider adding a discord integration by posting the resulting message via [Discord webhook](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks)

**Comment by jezevec10** [comment]
I'm thinking in which way to display the output in a Discord message to make it well readable..

Here I tried formatting in monospace font as code block, compared to normal text message
![image](https://user-images.githubusercontent.com/7414891/99755299-b7d78200-2aea-11eb-8e11-8e8fa4967f43.png)

But it's just an idea. Maybe it would be best to just generate a png image to get most precise formatting as a table with green and red arrows showing leaderboard direction change, but that's probably getting too complex which may not be worth it.

**Comment by bhainesva** [comment]
Thanks for testing it out @jezevec10. I've added a config file with a `discordWebhook` property. I tweaked the output a little too, here's what it's looking like currently. 
![image](https://user-images.githubusercontent.com/6308571/99779173-abe7c200-2ae2-11eb-9cec-91d618e85469.png)

Let me know if it works for you! The image idea sounds good but that's probably too complicated for me to figure out in the near future.

**Comment by jezevec10** [comment]
The formatting you came up with is awesome! I dont think the image idea is needed at all with how this is formatted now, it looks pretty good.

![image](https://user-images.githubusercontent.com/7414891/99809528-b2a62180-2b42-11eb-9f70-01c643a1fa49.png)
Maybe I'm not zoomed enough or the arrow shows thinner on my end. I proposed a change in #3 to replace the arrows with updown triangles (▼ / ▲) which should have better visibility even on zoomed-out screen.

I've installed the script to post updates in #bot-achievements in jstris discord, will let you know in case I think of other ideas for improvement, but so far it seems very good, thanks for making it ^^
